types/react-native/index.d.ts

```diff
@@ -9037,7 +9039,7 @@ export namespace Animated {
     };
 
     export interface AnimatedComponent<T extends React.ComponentType<any>>
-        extends React.FC<AnimatedProps<React.ComponentPropsWithRef<T>>> {}
+        extends React.SFC<AnimatedProps<React.ComponentPropsWithRef<T>>> {}
```

Comment:
I think this type is deprecated. What motivated this change?

Edit:
Is this motivated by
> Currently, it's just not possible to use Animated.View component in the code.

?

How did you try to use `Animated.View` and what happened specifically?

Reply:
I use it as
```
<Animated.View style={ ...some style }>
  { children }
</Animated.View>
```
And I got this error `JSX element type 'Animated.View' does not have any construct or call signatures.`

Ok, now I see that the problem is not with the type, but mostly with a package.json. It define `@types/react` dependency with any version, although `React.FC` wasn't exist in `@types/react` version 16.3.0 (which was stick in my yarn.lock file). Probably it could make sense to limit react version (especially that react-native itself also need specific version of react, for intance react-native 66 requires react 17)